README.md

```diff
@@ -166,7 +166,7 @@ Unlike testing and web scraping libraries you're used to, Panther:
 * supports custom [Selenium server](https://www.seleniumhq.org) installations
 * supports remote browser testing services including [SauceLabs](https://saucelabs.com/) and [BrowserStack](https://www.browserstack.com/)
 
-### Using the `ServerListener` to Always Have a Running Web Server
+### Using the `ServerListener`/`ServerExtension` to Always Have a Running Web Server
```

Comment:
If we want to recommend 2 tools for different versions, I'd update this for the following:

```
Improve performances by having a persistent web server running
```

Reply:
Any suggestion to reword the body of this chapter ?

Reply:
Yes:
(sorry couln't use a diff because I don't know how to handle code blocks inside code blocks :rofl: )

---

When you use the Panther client, the web server running in background will be started at runtime and stopped at test's
teardown.

If you want to improve performances and launch the server at test suite startup, you can hook to PHPUnit to
your `phpunit.xml.dist` configuration file with Panther's listeners:

```xml
<!-- phpunit.xml.dist -->

    <!-- PHPUnit <7.3 -->
    <listeners>
        <listener class="Symfony\Component\Panther\ServerListener" />
    </listeners>

    <!-- PHPUnit 7.3+ -->
    <extensions>
        <extension class="Symfony\Component\Panther\ServerExtension" />
    </extensions>
```

The listener will start the webserver when the test suite is started, and will stop it when all your tests are executed.

Reply:
Added.